payjoin/src/receive/error.rs

```diff
@@ -41,7 +41,9 @@ pub struct RequestError(InternalRequestError);
 #[derive(Debug)]
 pub(crate) enum InternalRequestError {
     Psbt(bitcoin::psbt::Error),
+    PsbtParseError(bitcoin::psbt::PsbtParseError),
     Base64(bitcoin::base64::DecodeError),
+    PsbtFinalizingError(&'static str),
```

Comment:
This always returns the same kind of string, so it doesn't need to take a value at all

`InternalRequestError::PsbtFinalize` would be fine

Reply:
fixed!